### PR TITLE
Stop building dotnet-watch in AspNetCore

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Review by captainsafia — APPROVED** [review]
I love the smell of deleted code in the morning.

File: AspNetCore.sln [hunk 1]
```diff
@@ -1235,12 +1235,6 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Extensions.FilePr
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Tools", "Tools", "{0B200A66-B809-4ED3-A790-CB1C2E80975E}"
 EndProject
-Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "dotnet-watch", "dotnet-watch", "{B6118E15-C37A-4B05-B4DF-97FE99790417}"
```
Comment (captainsafia) [inline AspNetCore.sln:1238]: You might also want to nuke it from the *.slnf files.

Comment (pranavkm) [reply AspNetCore.sln:1238]: Done

**Comment by pranavkm** [comment]
@wtgodbe do we wait for 5.0 branches to open to get this change in? It's all infrastructural.

**Comment by wtgodbe** [comment]
> do we wait for 5.0 branches to open to get this change in? It's all infrastructural.

Yep, gotta wait until branches are open. The issue isn't what kind of changes go in, it's the fact that new commits trigger new builds which publish new things with the old branding (and, if we aren't careful or the timing is wrong, new builds trigger dependency flow, which compounds the problem).

**Review by mkArtakMSFT — APPROVED** [review]
Thank you!

**Pull request merged**